configs/opencv-ovms/gst/framework-pipelines/yolov5_pipeline/yolov5s.sh

```diff
@@ -9,10 +9,15 @@ DECODE="${DECODE:="decodebin force-sw-decoders=1"}" #decodebin|vaapidecodebin
 DEVICE="${DEVICE:="CPU"}" #GPU|CPU|MULTI:GPU,CPU
 PRE_PROCESS="${PRE_PROCESS:=""}" #""|pre-process-backend=vaapi-surface-sharing|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 
 AGGREGATE="${AGGREGATE:="gvametaaggregate name=aggregate !"}" # Aggregate function at the end of the pipeline ex. "" | gvametaaggregate name=aggregate
-OUTPUTFORMAT="${OUTPUTFORMAT:="! fpsdisplaysink video-sink=fakesink sync=true --verbose"}" # Output format for the pipeline  "! fpsdisplaysink video-sink=fakesink sync=true --verbose" | (render_mode)"! videoconvert ! video/x-raw,format=I420 ! gvawatermark ! videoconvert ! fpsdisplaysink video-sink=ximagesink sync=true --verbose"
+
+if [ "$RENDER_MODE" == "1" ]; then
+    OUTPUT="${OUTPUT:="! videoconvert ! video/x-raw,format=I420 ! gvawatermark ! videoconvert ! fpsdisplaysink video-sink=ximagesink sync=true --verbose"}"
```

Comment:
where is the default value of ${OUTPUT} defined? should we add that into env files?

Reply:
I don't know what the default value should be. Maybe the value of render_mode=0

Reply:
Default output are defined in the .env files https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/envs/yolov5-cpu.env#L13
https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/envs/yolov5-gpu.env#L12

Reply:
It actually makes sense to move it to the script and out of the .env since the format is related to render mode and not the target_device.

Reply:
ok, do you mean I should delete that line in the env file? 
And that's why it wasn't working correctly, that's the reason I changed the variable name. It was conflicting with the env var default.

Reply:
Yes, you can delete those lines since we no longer use them.

Reply:
i agree with Brian, we should remove `OUTPUTFORMAT` from env files if it is not referenced anywhere in the code.  i was originally suggested to add `OUTPUT` you created here into env files for definition with default value; but Brian is right on this is only RENDER_MODE related logic so we use RENDER_MODE env to control this from the end user pov.